src/scheduler/job_info.c

```diff
@@ -2835,6 +2854,7 @@ dup_job_info(job_info *ojinfo, queue_info *nqinfo, server_info *nsinfo)
 	njinfo->array_index = ojinfo->array_index;
 	njinfo->array_id = string_dup(ojinfo->array_id);
 	njinfo->queued_subjobs = dup_range_list(ojinfo->queued_subjobs);
+	njinfo->max_run_subjobs = ojinfo->max_run_subjobs;
```

Comment:
Well in case of subjobs it will be 0 and for parent jobs it will be set and get copied.
running_subjobs is calculated in dup_server_info because we need to associate the parent job pointer as well. So associate_parent_job() function increments running_subjobs as well.